src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -920,14 +920,20 @@ def __call__(self, result):
             result = super(DeploymentOutputLongRunningOperation, self).__call__(result)
             outputs = None
             try:
-                outputs = result.properties.outputs
+                if isinstance(result, str) and result:
+                    try:
+                        obj = json.loads(result)
+                        return obj
+                    except json.decoder.JSONDecodeError:
+                        logger.info("Fail to transform result \"%s\" to dictionary", result)
```

Comment:
https://docs.python.org/3/library/string.html#string-formatting
This is new style. More fashionable.